dist/bin/dotd

```diff
@@ -75,4 +75,10 @@ CLASS_PATH="$CLASS_PATH$PSEP$SNAKEYAML_LIB"
 CLASS_PATH="$CLASS_PATH$PSEP$ST4_LIB"
 CLASS_PATH="$CLASS_PATH$PSEP$JSOUP_LIB"
 
-eval exec "\"$JAVACMD\"" $CLASS_PATH dotty.tools.dottydoc.Main $@
+eval exec "\"$JAVACMD\"" \
+    -Dscala.usejavacp=true \
+    $CLASS_PATH \
+    dotty.tools.dottydoc.Main \
+    $@
```

Comment:
This change looks good, in line 67, the dot(`.`) is not actually required, dottydoc doesn't dependent on custom code to run. It's automatically added if user doesn't pass an explicit `-classpath`.